draft-ietf-dnsop-dns-catalog-zones.md

```diff
@@ -204,11 +207,22 @@ for authoritative nameserver management only and are not intended for general
 querying via recursive resolvers.
 
 
-## Global Properties
+## Properties
 
 Apart from catalog zone metadata stored at the apex (NS, SOA and the like), catalog zone information is stored in the form of "properties".
 Catalog consumers SHOULD ignore properties they do not understand.
 
+Properties have a name combined with an associated RR type.
+Properties that have a name known to a catalog consumer, but the wrong RR type,
+SHOULD be ignored.
```

Comment:
Implementors may specify properties that require several record sets of different types. The following suggestion covers this:

```suggestion
Properties are identified by their name, which is used as an owner name prefix for one or more record sets underneath a member node, with type(s) as appropriate for the respective property.
Record sets that appear at a property owner name known to the catalog consumer, but with the wrong RR type, invalidate the property as a whole.
The consumer SHOULD ignore such invalid properties as a whole.
```

Reply:
This is better, but I don't like the word `invalidate` (or invalid), because invalid properties are such, that MUST kill the whole catalog processing. Unknown RRs just are to be ignored. And IMHO, if it appears in a way that there is a known goor property, plus some additional unknow RR types, it MUST NOT invalidate the property. I would vote for simply ignoring unknown RR (types) in any occasion, and continuing as if they weren't there:
```
Properties are identified by their name, which is used as an owner name prefix for one or more record sets underneath a member node, with type(s) as appropriate for the respective property.
Record sets that appear at a property owner name known to the catalog consumer, but with another RR type, SHOULD be ignored by the consumer.
```

Reply:
I agree with libor on the suggested behavior and naming such records something else than "invalid" ("unknown" makes a lot of sense to me). I would use that instead of "another RR type".

```
Properties are identified by their name, which is used as an owner name prefix for one or more
record sets underneath a member node, with type(s) as appropriate for the respective property.
Record sets that appear at a property owner name known to the catalog consumer but with an
unknown RR type, SHOULD be ignored by the consumer.
```